src/main/java/uk/gov/hmcts/reform/professionalapi/service/impl/OrganisationServiceImpl.java

```diff
@@ -1056,5 +1057,31 @@ private boolean getMoreAvailable(Page<Organisation> pageableOrganisations) {
         return !pageableOrganisations.isLast();
     }
 
+    @Override
+    public ResponseEntity<ContactInformationResponse> updateContactInformationForOrganisation(
+        ContactInformationCreationRequest contactInfo, String organisationIdentifier) {
+
+        Organisation organisation = organisationRepository.findByOrganisationIdentifier(organisationIdentifier);
+        List<ContactInformation> existingContactInformationList = organisation.getContactInformation();
+        //if multiple contact informations are found they will all be updated
+        if (contactInfo != null) {
```

Comment:
If contact information is not found then return not found response otherwise update

Reply:
added check if contact information not found then showing resource not found error